sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelects.java

```diff
@@ -12,7 +12,7 @@
  * @see SettingSelector
  * @see ConfigurationAsyncClient
  */
-public enum SettingFields {
+public enum SettingSelects {
```

Comment:
Maybe name this to `FieldSelections`? We should bring this naming up in the stand-up

Reply:
There's a teams discussion in the azconfig channel about this. I think the name should be agreed on by all language folks before updating.